lib/editor/util/gtfs.js

```diff
@@ -1,37 +1,88 @@
 // @flow
 
-import along from 'turf-along'
-import lineDistance from 'turf-line-distance'
 import {latLngBounds} from 'leaflet'
+// import along from '@turf/along'
+// import lineString from 'turf-linestring'
+// import lineDistance from 'turf-line-distance'
 
-import {newControlPoint} from './map'
+// import {newControlPoint} from './map'
 import type {
-  ControlPoint,
+  // ControlPoint,
   Entity,
   GtfsRoute,
   GtfsStop,
   Pattern,
-  ServiceCalendar,
-  StopTime
+  ServiceCalendar // ,
+  // PatternStop
 } from '../../types'
 
-export const componentList: Array<string> = [
-  'route',
-  'stop',
-  'fare',
-  'feedinfo',
-  'calendar',
-  'scheduleexception',
-  'agency'
+// export const parseActiveFromRouteParams = (routeParams) => {
+//   // TODO: Figure out a better way to parse an integer from react-router route
+//   // params.
+//   const activeEntityId = +routeParams.activeEntityId
+//   const subEntityId = +routeParams.subEntityId
+//   return {
+//     ...routeParams,
+//     activeEntityId,
+//     subEntityId
+//   }
+// }
+
+// FIXME: Add flow.  Consolidate with gtfs/util/getEntityGraphQLRoot
+export const COMPONENT_LIST = [
```

Comment:
Use an object for this if it is needed.  There'd be no need to do array searches.